src/components/SystemMetrics.jsx

```diff
@@ -284,7 +284,7 @@ function Graph({ title }) {
         ).format("HH:mm")})`,
       );
       setData(historical_data);
-    } else if (title === "ALGOs Locked") {
+    } else if (title === "ALGOs Locked in v1") {
```

Comment:
I'm not convinced pointing out that these are v1 metrics is better than hiding the page unless we can quickly rollout the same for v2